examples/undocumented/python_modular/graphical/interactive_svm_demo.py

```diff
@@ -132,7 +129,7 @@ def train_svm(self):
         svm.set_epsilon(1e-2)
 
         x, y, z = util.compute_output_plot_isolines(svm, gk, train)
-        plt=self.axes.pcolor(x, y, z, shading='interp')
```

Comment:
`shading` is deprecated in `matplotlib` so removing this parameter. No visual changes in the plots.
```
/usr/local/lib/python2.7/dist-packages/matplotlib/cbook.py:137: MatplotlibDeprecationWarning: The shading option was deprecated in version 1.2. Use edgecolors instead.
  warnings.warn(message, mplDeprecation, stacklevel=1)
```